chapters/project-guide/project-guide.rst

```diff
@@ -0,0 +1,28 @@
+##################
+Project set up guide
+##################
+
+You will see the best practices on how to create a project on Jina.
+
+
+*********
+Feature description and expected outcome
```

Comment:
Just to make sure @FionnD , the template should be something like this right?

Reply:
Yup